www/include/common/javascript/datepicker/localizedDatepicker.js

```diff
@@ -105,6 +105,9 @@ function setUserFormat() {
                     jQuery('<script>')
                         .attr('src', './include/common/javascript/datepicker/' + data)
                         .appendTo('body');
+                    jQuery('<script>')
```

Comment:
the ajax will check if the datepicker library exists for the chosen langage, not the timepicker one.
We shouldn't try to load the timepicker lib from here

Reply:
or you should modify this file and include the check for the timepicker lib
you may need to rename the class name and the filename too

Reply:
to stay consitent, you need to change the files timepicker localization names

Reply:
thanks @sc979 I will create a new class with new name to use with timepicker and don't mix the libs

Reply:
if you modify the called class (by the ajax), it will avoid to call twice an ajax request ;)
but rename it to stay  consistent with what it does. This class is used only here